src/BenchmarksServer/Startup.cs

```diff
@@ -217,6 +221,28 @@ public static int Main(string[] args)
                                 Debug.Assert(process == null);
                                 process = StartProcess(hostname, database, sqlConnectionString, Path.Combine(tempDir, benchmarksDir),
                                     job, dotnetHome);
+                                
+                                if (timer != null)
+                                {
+                                    timer.Dispose();
+                                }
+
+                                DateTime lastMonitorTime = DateTime.UtcNow;
+                                var oldCPUTime = new TimeSpan(0);
+
+                                timer = new Timer(_ => 
+                                {
+
+                                    var newCPUTime = process.TotalProcessorTime;
+                                    var ellapsed = DateTime.UtcNow.Subtract(lastMonitorTime).TotalMilliseconds;
```

Comment:
Typo `ellapsed` -> `elapsed`

Reply:
NIT: I would either change the name to `elapsedMilliseconds`, or else keep the type as `TimeSpan` and only convert to milliseconds when computing `cpu`, like you do with `newCPUTime` and `oldCPUTime`.